packages/slice-machine/components/StatusBadge/index.tsx

```diff
@@ -30,7 +30,6 @@ export const StatusBadge: React.FC<StatusBadgeProps> = ({
     <>
       <Text data-for={`${modelId}-tooltip`} data-tip>
         <Badge
-          mr="2"
```

Comment:
UI team is going to have fun with this 😅

Reply:
ahah why?

Reply:
margins right?